core/src/sigverify_stage.rs

```diff
@@ -254,27 +300,40 @@ impl SigVerifyStage {
         let mut last_print = Instant::now();
         Builder::new()
             .name("solana-verifier".to_string())
-            .spawn(move || loop {
-                if let Err(e) =
-                    Self::verifier(&packet_receiver, &verified_sender, &verifier, &mut stats)
-                {
-                    match e {
-                        SigVerifyServiceError::Streamer(StreamerError::RecvTimeout(
-                            RecvTimeoutError::Disconnected,
-                        )) => break,
-                        SigVerifyServiceError::Streamer(StreamerError::RecvTimeout(
-                            RecvTimeoutError::Timeout,
-                        )) => (),
-                        SigVerifyServiceError::Send(_) => {
-                            break;
+            .spawn(move || {
+                let mut bloom = Bloom::random(1_000_000, 0.0001, 1 << 20).into();
+                let mut bloom_age = Measure::start("bloom_age").as_ms();
+                loop {
+                    let now = Measure::start("bloom_age").as_ms();
+                    if now - bloom_age > 1_000 {
```

Comment:
1s seems too fast for refresh, it will still let dupes in once per second which could double the load during the approx 2 seconds of 4 consecutive leader slots. How about 10s?

Reply:
maybe refresh every 2sec for leader slots?

Reply:
it seems dangerous for it to be perfect.  so it will let the remote node retry.  2s seems like a reasonable retry time though.

Reply:
or 5 or 10, not sure how to pick one here.  we can let the stats guide us once we get some numbers

Reply:
Yeah I suppose one risk is that a client could be sending to tpu expecting you to forward to the right machine and you forward to someone who doesn't create a block. The client asks again and you just drop the packet rather than forwarding to the new leader. But this assumes clients rely on a static node to forward transactions for them, which I don't think is the case, though, I think clients try resending to different leaders so I think being more aggressive is fine here.

Reply:
2 seconds seems reasonable, since it covers the leaders slot